ui/src/components/Flags/Flag.vue

```diff
@@ -159,6 +167,12 @@
                   placeholder="Description"
                 />
               </b-field>
+              <b-field label="Attachment (optional)">
+                <b-input
+                  v-model="newVariant.attachment"
+                  placeholder="Attachment"
```

Comment:
```suggestion
                <b-input
                  v-model="newVariant.attachment"
                  maxlength="255"
                  type="textarea"
                  placeholder="{}"
```

Reply:
@markphelps shouldn't `maxlength` be `10000` if that's the max attachment size?

Reply:
yeah, it should likely be larger than 255, that was my fault. I just didn't want the UI to look crazy when displaying the attachment in the table after creation. Perhaps we should remove the `attachment` in the list view if we are going to be displaying the entire payload, or just truncate it after a certain number of characters?

I'm talking about this view:

![152921355-0b70fd73-e1b9-4f2c-b6e0-fd1b5d5cba1b](https://user-images.githubusercontent.com/209477/153001997-d9d070a0-39e5-433d-8628-3f46a8653c21.png)

Reply:
Other than removing the `attachment` column, I think there's 2 other options:

1. Change it to be a 'Has Attachment' and show a checkbox if so, like we do [here](https://github.com/markphelps/flipt/blob/f41aff89f6a7b9948eb9035e4c20153d2c54beb3/ui/src/components/Flags.vue#L54-L56)
2. Use the ` | limit` [filter](https://github.com/markphelps/flipt/blob/f41aff89f6a7b9948eb9035e4c20153d2c54beb3/ui/src/main.js#L14) which only shows the first 30 characters like we do [here](https://github.com/markphelps/flipt/blob/f41aff89f6a7b9948eb9035e4c20153d2c54beb3/ui/src/components/Flags.vue#L58) 

I kind of like the 1st option, but would like to here what you all think

Reply:
I kind of like 2nd option as I can have a quick overview of all the attachments. I can then click "edit" on the variant of interest and see the entire attachment.